pubspec.yaml

```diff
@@ -10,6 +10,7 @@ repository: https://github.com/primequantuM4/pixel_prompt
 issue_tracker: https://github.com/primequantuM4/pixel_prompt/issues
 
 dependencies:
+  dartdoc: ^8.3.4
```

Comment:
Don't need to add in dependencies, 'dev_dependencies' if its needed but I think if you publish your package, pub.dev automatically takes care of building the documentation